model_analyzer/record/record.py

```diff
@@ -106,13 +107,22 @@ def __init__(self, value, timestamp):
         self._value = value
         self._timestamp = timestamp
 
+    @staticmethod
+    def aggregation_function():
```

Comment:
It would be great if this can be one of the constructor's arguments with the default that you have here.

Reply:
Addressed below